tests/s3api_test.go

```diff
@@ -373,30 +412,39 @@ func OSReadDir(root string) ([]string, error) {
 }
 
 func call(method string, path string, node citools.Node, body io.Reader) (respBody string, statusCode int, err error) {
```

Comment:
1. why is this necessary? Doesn't that indicate a flake we should investigate (retrying)
2. Can you rename this function to indicate that it's retried?

Reply:
There's a code path in the S3 handlers that return that code with a message to please try again, as the FSM hasn't returned to the active state yet, so I made sure the client tried again in that case. It cropped up in the test suite, which of course fires requests at the FSMs without any delays, so I reckon it could turn up in real life too!

And sure, I'll make it clear in the function name!